pkg/build/release_push_build.go

```diff
@@ -20,11 +20,13 @@ import (
 	"fmt"
 	"regexp"
 
-	"sigs.k8s.io/kubetest2/pkg/exec"
+	"github.com/pkg/errors"
+	"k8s.io/release/pkg/build"
```

Comment:
probably worth aliasing this since this package is also called build

Reply:
Done :+1: